src/api/eventsAPI.js

```diff
@@ -1,12 +1,18 @@
 import axios from 'axios';
 
-function getEvents({ page = 0, perPage = 25 }) {
-  return axios.get(`https://resistance-calendar.herokuapp.com/v1/events?page=${page}&per_page=${perPage}`)  // eslint-disable-line max-len
+import { queryBuilder } from '../utils';
+
+const BASE_URL = 'https://resistance-calendar.herokuapp.com/v1/events';
+
+function getEvents({ page = 0, perPage = 25 }, filterParams) {
+  const filterUrl = queryBuilder.eventsFilter(filterParams);
+
+  return axios.get(`${BASE_URL}?page=${page}&per_page=${perPage}${filterUrl}`)
```

Comment:
Just thinking in a separate PR we should create a issue relating to sorting. I imagine we sort by date but what happens when two event have the same date? Are we using the time of the event or distance to user?